Makefile

```diff
@@ -226,7 +227,7 @@ travis-cover: btcd unit-cover goveralls
 
 flakehunter: build-itest
 	@$(call print, "Flake hunting ${backend} integration tests.")
-	while [ $$? -eq 0 ]; do $(ITEST); done
+	while [ $$? -eq 0 ]; do make itest icase='${icase}' backend='${backend}'; done
```

Comment:
shouldn't this inner match the script in `itest-only` above?

Reply:
Ah yes, that should speed up things a bit.